index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>Home &mdash; Hannah Lumapas</title>
+        <link rel="stylesheet" href="./stylesheets/style.css" />
+
+        <!-- =====BOX ICONS==== -->
+        <link href="https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css" rel="stylesheet">
```

Comment:
I like how you explored other stylesheets out there however you can totally make this feature using the  `a:active` pseudo class.